Library/Formula/libwandevent.rb

```diff
@@ -0,0 +1,12 @@
+class Libwandevent < Formula
+  desc "Libwandevent provides an API for developing event-driven programs"
+  homepage "http://research.wand.net.nz/software/libwandevent.php"
+  url "http://research.wand.net.nz/software/libwandevent/libwandevent-3.0.1.tar.gz"
+  sha256 "317b2cc39f912f8e5875b9dd05658cd48ead98bf20a1d89855e5a435381bee24"
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+end
```

Comment:
Could you please a test? See [`tinyxml2.rb`](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/tinyxml2.rb#L22) for an example of a library formula with a good test.